assets/js/app/llms-forms.js

```diff
@@ -396,7 +396,7 @@ LLMS.Forms = {
 	 * @return {string} The placeholder text.
 	 */
 	get_state_placeholder: function( holder, country_code ) {
-		var placeholder = holder.attr('placeholder').replace('{REGION}',country_code[country_code]?.state ?? "DEFAULT");
+		var placeholder = holder.attr( 'placeholder' ).replace( '{REGION}' ,country_code[country_code]?.state ?? "DEFAULT" );
```

Comment:
in this update the `country_code` variable is a string (not the actual object of data) so this isn't going to work as expected.

Additionally I'm realizing now we have to specify a default value, but specifying the string `DEFAULT` will cause problems.

We'll need to use the `LLMS.l10n.translate()` function to translate the default, which should be "State / Region"